# HG changeset patch
# User Rik <rik@octave.org>
# Date 1401669233 25200
#      Sun Jun 01 17:33:53 2014 -0700
# Node ID 96527fbdb8d44b139a52a79203f0e904982cac09
# Parent  ff613b6a7b6f780b7197f2e2bbe5dbae81c409f9
edit.m: Remove keyboard() statement left from debugging cset 0728f7b6b1a2.

* edit.m: Remove keyboard statement left over from debugging.

diff --git a/scripts/miscellaneous/edit.m b/scripts/miscellaneous/edit.m
--- a/scripts/miscellaneous/edit.m
+++ b/scripts/miscellaneous/edit.m
@@ -344,17 +344,16 @@ function ret = edit (varargin)
       name = file(1:idx-1);
       ext = file(idx+1:end);
     else
       name = file;
       ext = "";
     endif
     if (! any (strcmp (ext, {"cc", "m"})))
       ## Some unknown file.  Create and open it or just open it.
-      keyboard;
       if (isempty (ext))
         fileandpath = [fileandpath ".m"];  # Add .m extension per default
       endif
       if (isguirunning ())
         ## Write the initial file (if there is anything to write)
         ## Give user the opportunity to change the file extension
         fileandpath = uiputfile (fileandpath);
         fid = fopen (fileandpath, "wt");
